tests/functional/web/form_checkbox_radio.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Form</title>
+</head>
+<body>
+<form id="another-form" method="POST">
+    <input name="j1">
+    <input type="checkbox" name="j2" value="j2a">
+
+    <label>
+        J2B
```

Comment:
To make sure the normalize-space() selectors work as expected, there could also be a case where the visible text has multiple spaces inside, ie. not being just one word, like:
 <pre>
<label>
   J     2 

  B    
...
 </pre>

What are you thoughts?